### PR TITLE
Externally managed content update

### PR DESCRIPTION
**Pull request by twhetzel**
Update the externally managed content. This includes all content, including updates to the rare disease subsets.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -53127,10 +53125,8 @@ synonym: "skin metatypical basal cell carcinoma" EXACT [DOID:4281]
 synonym: "skin metatypical carcinoma" EXACT [DOID:4281]
 xref: DOID:4281 {source="MONDO:equivalentTo"}
 xref: ICDO:8095/3 {source="NCIT:C66903"}
-xref: MEDGEN:362831 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: NCIT:C66903 {source="MONDO:equivalentObsolete"}
 xref: SCTID:254702000 {source="MONDO:equivalentTo"}
-xref: UMLS:C1883040 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:362831"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:53133]: Confirmed this UMLS concept was removed: C1883040 last appeared in the 2024AA version of the UMLS. It was mapped to a broader CUI: [C4721806](https://uts.nlm.nih.gov/uts/umls/concept/C4721806).

**Comment by twhetzel** [comment]
I spot checked a few changes and they match what the source has. There is an issue with the content of the NORD file for some synonym content so synonyms are being added, but do not have the NORD provenance, e.g, 
```
synonym: "Physical urticaria" EXACT []
synonym: "Levy-Yeboa Syndrome" EXACT []
synonym: "RYR1-Related Diseases" EXACT []
```

I will create a ticket to outline the problem so that the file can be fixed/updated by the data contributor.

**Review by sabrinatoro — APPROVED** [review]
Looks good to me. 
There are a lot of new MedGen/UMLS x-refs. I am assuming it is reflected in the increased side of the file we got from them. 
I noted we have a few addition of synonyms which are added without a x-ref. I don't think this necessitate delaying this PR merge, but we should investigate it.
@twhetzel, feel free to merge after you have seen this comment.  Thank you

**Comment by twhetzel** [comment]
Thanks @sabrinatoro I spot checked a few of the MedGen/UMLS x-refs (well, for MedGen where the site would load) and the changes represent the source as best I can see. 

I did mention the missing synonym xrefs [here](https://github.com/monarch-initiative/mondo/pull/8499#issuecomment-2552037571), but the information is missing in the source file and I will create a ticket and contact NORD to sort this out for the next round.

**Pull request merged**